adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/CassandraStorageOperations.java

```diff
@@ -55,14 +55,18 @@ public class CassandraStorageOperations implements StorageOperations
      */
     public CassandraStorageOperations(JmxClient jmxClient, DnsResolver dnsResolver)
     {
-        this(jmxClient, new RingProvider(jmxClient, dnsResolver));
+        this(jmxClient,
+             new RingProvider(jmxClient, dnsResolver),
+             new TokenRangeReplicaProvider(jmxClient, dnsResolver));
     }
 
-    public CassandraStorageOperations(JmxClient jmxClient, RingProvider ringProvider)
+    public CassandraStorageOperations(JmxClient jmxClient,
```

Comment:
I wonder why we have this constructor. It only seems to be used from the call in line 58. Should we remove it?